src/updaters/php/root-composer-update-packages.ts

```diff
@@ -26,11 +26,17 @@ export class RootComposerUpdatePackages extends DefaultUpdater {
    * @returns {string} The updated content
    */
   updateContent(content: string, logger: Logger = defaultLogger): string {
-    if (!this.versionsMap || this.versionsMap.size === 0) {
+    if (!this.version && (!this.versionsMap || this.versionsMap.size === 0)) {
       logger.info('no updates necessary');
       return content;
     }
     const parsed = JSON.parse(content);
+    if (parsed['version']) {
+      const fromVersion: string = parsed['version'];
+      const toVersion = this.version.toString() || '1.0.0';
+      parsed['version'] = toVersion;
+      logger.info(`updating "version" from ${fromVersion} to ${toVersion}`);
+    }
```

Comment:
Can you add tests for setting this root `version` entry (with the value set and unset)?

Reply:
I believe this is the kind of test you're looking for? https://github.com/googleapis/release-please/pull/2201/commits/5093438d6e8b111f4a8a75213109b641c58d93fd